resources/assets/components/Action/index.js

```diff
@@ -86,7 +86,7 @@ class Action extends React.Component {
             </li>
           </ul>
         </div>
-        {new Date(format(parse(campaign.start_date), 'M/d/YY')) > Date.now() ? (
+        {new Date(parse(campaign.start_date)) > Date.now() ? (
```

Comment:
@chloealee Can you try the snippet I'd posted in [my comment](
https://github.com/DoSomething/rogue/pull/861#discussion_r267789300)? The [`parse`](https://date-fns.org/v1.30.1/docs/parse) function returns a `Date`, so there shouldn't be a reason to re-instantiate it by passing the result into the `Date` constructor:

```js
parse(campaign.start_date) > Date.now()
```

Reply:
@DFurnes oh sorry I wasn't looking close enough! Updated that!